packages/cli/src/info/cdk_info.ts

```diff
@@ -0,0 +1,63 @@
+import { execa as _execa } from 'execa';
+import { printer } from '../printer.js';
+
+/**
+ * Provides information about CDK.
+ */
+export class CdkInfoProvider {
+  /**
+   * Format CDK information.
+   * @param execa - The execa to run the cdk command.
+   * @returns The cdk doctor output.
+   */
+  async getCdkInfo(execa = _execa): Promise<string> {
```

Comment:
if this is returning `string` then could we please make `formatCdkInfo` private and return formatted output from here?
Is there any case we'd want unformatted output somewhere ?

Reply:
Also, execa should be injected into the ctor rather than the method (it's fine to still have the default value since it's only used for testing)